apps/web/vibes/soul/primitives/products-list/index.tsx

```diff
@@ -21,6 +21,7 @@ interface Props {
   compareParamName?: string;
   emptyStateTitle?: string;
   emptyStateSubtitle?: string;
+  count?: number;
```

Comment:
We should rename this to `skeletonCardCount`

Taking a step back, though, should this even be configurable?

Reply:
You know, this value does not need to change. I can't find an example right now where it makes sense to make it configurable.

Reply:
I agree. I think we can keep it non-configurable for now